packages/react-instantsearch-core/src/core/createInstantSearchManager.js

```diff
@@ -364,8 +364,8 @@ export default function createInstantSearchManager({
       };
     }
 
-    if (Array.isArray(results)) {
-      hydrateSearchClientWithMultiIndexRequest(client, results);
+    if (Array.isArray(results.results)) {
```

Comment:
This duplication reflects that the naming is wrong, perhaps we should go with `stateResult.results` (like we return `{results} = await findResultState()`.

Reply:
ah, I didn't think of renaming the variable, that would be a good idea in the future